src/hooks/useFirestore.js

```diff
@@ -11,11 +11,12 @@ const useFirestore = (collection) => {
         documents.push({ ...doc.data(), id: doc.id });
       });
       setDocs(documents);
-      return () => {
-        unsubscribe();
-      };
     });
+    return () => {
```

Comment:
Return was incorrectly placed inside of the function passed to onSapshot. Moved to be returned from useEffect to unsubscribe when unmounting component.